src/explore-education-statistics-frontend/src/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
[746,1],[1013,1],[1009,1],[1012,1],[1010,1],[271,1],[649,1],[270,1],[1014,1],[1015,1],[1016,1],[1017,1],[1023,1],[1018,1],[1022,1],[1019,1],[1021,1],[267,1],[272,1],[1024,1],[1025,1],[1026,1],[1027,1],[1028,1],[1029,1],[1037,1],[1036,1],[1035,1],[1020,1],[1038,1],[715,1],[690,1],[719,1],[842,1],[716,1],[720,1],[884,1],[1040,1],[1041,1],[268,1],[1042,1],[103,1],[60,1],[104,1],[61,1],[62,1],[63,1],[64,1],[65,1],[66,1],[67,1],[68,1],[69,1],[70,1],[71,1],[72,1],[59,1],[100,1],[73,1],[74,1],[75,1],[105,1],[76,1],[77,1],[78,1],[79,1],[80,1],[81,1],[82,1],[83,1],[84,1],[85,1],[86,1],[87,1],[88,1],[89,1],[90,1],[91,1],[99,1],[102,1],[92,1],[93,1],[94,1],[95,1],[96,1],[101,1],[97,1],[98,1],[1043,1],[1044,1],[1045,1],[50,1],[266,1],[265,1],[1046,1],[107,1],[764,1],[1047,1],[1048,1],[48,1],[52,1],[1049,1],[1050,1],[51,1],[269,1],[1051,1],[1075,1],[1076,1],[1077,1],[1039,1],[1079,1],[1078,1],[1080,1],[1081,1],[777,1],[731,1],[49,1],[264,1],[1071,1],[1070,1],[1069,1],[1068,1],[776,1],[1072,1],[837,1],[1030,1],[1032,1],[1034,1],[1033,1],[1031,1],[188,1],[191,1],[187,1],[190,1],[189,1],[58,1],[161,1],[163,1],[121,1],[131,1],[122,1],[123,1],[132,1],[112,1],[172,1],[174,1],[156,1],[155,1],[154,1],[177,1],[153,1],[180,1],[108,1],[110,1],[160,1],[164,1],[168,1],[115,1],[114,1],[135,1],[143,1],[118,1],[113,1],[109,1],[147,1],[146,1],[136,1],[106,1],[144,1],[137,1],[145,1],[148,1],[124,1],[138,1],[119,1],[142,1],[141,1],[120,1],[134,1],[133,1],[125,1],[126,1],[128,1],[127,1],[129,1],[179,1],[130,1],[57,1],[158,1],[166,1],[170,1],[150,1],[111,1],[152,1],[157,1],[140,1],[139,1],[117,1],[116,1],[162,1],[159,1],[47,1],[56,1],[53,1],[54,1],[55,1],[165,1],[167,1],[169,1],[171,1],[184,1],[173,1],[183,1],[175,1],[151,1],[176,1],[178,1],[181,1],[182,1],[149,1],[273,1],[1074,1],[745,1],[744,1],[262,1],[740,1],[1073,1],[2,1],[3,1],[4,1],[5,1],[6,1],[7,1],[8,1],[9,1],[834,1],[192,1],[653,1],[738,1],[652,1],[741,1],[650,1],[660,1],[654,1],[655,1],[656,1],[657,1],[670,1],[658,1],[700,1],[742,1],[743,1],[659,1],[766,1],[767,1],[768,1],[274,1],[276,1],[186,1],[1097,1],[771,1],[770,1],[778,1],[835,1],[838,1],[829,1],[832,1],[833,1],[843,1],[844,1],[718,1],[692,1],[717,1],[845,1],[852,1],[888,1],[892,1],[893,1],[897,1],[898,1],[594,1],[595,1],[597,1],[706,1],[880,1],[701,1],[704,1],[709,1],[708,1],[691,1],[887,1],[883,1],[711,1],[705,1],[713,1],[714,1],[707,1],[899,1],[900,1],[901,1],[902,1],[903,1],[904,1],[721,1],[905,1],[907,1],[915,1],[917,1],[723,1],[914,1],[906,1],[913,1],[925,1],[724,1],[926,1],[927,1],[966,1],[967,1],[965,1],[963,1],[968,1],[969,1],[726,1],[185,1],[970,1],[973,1],[974,1],[975,1],[976,1],[977,1],[982,1],[983,1],[984,1],[986,1],[985,1],[987,1],[989,1],[990,1],[988,1],[991,1],[992,1],[994,1],[997,1],[998,1],[995,1],[996,1],[993,1],[978,1],[979,1],[980,1],[916,1],[999,1],[981,1],[1004,1],[1006,1],[1007,1],[972,1],[1005,1],[1000,1],[1003,1],[1001,1],[1002,1],[633,1],[634,1],[732,1],[728,1],[263,1],[730,1],[733,1],[729,1],[734,1],[735,1],[736,1]]},"version":"4.4.3"}
```

Comment:
Does this need to be .gitignored?

Reply:
Naughty!  I've removed it - it was an artifact from an upgrade branch that I must've accidentally included.